torch/quantization/fx/quantization_patterns.py

```diff
@@ -378,6 +378,27 @@ def convert(self, quantizer, node, load_arg, debug=False, convert_custom_config_
             load_arg(quantized=[0])(self.bn_node.args),
             load_arg(quantized=False)(self.bn_node.kwargs))
 
+@register_quant_pattern(torch.nn.Embedding)
+class Embedding(QuantizeHandler):
+    def __init__(self, quantizer, node):
+        super().__init__(quantizer, node)
+        assert node.op == 'call_module'
+        self.emb_node = node
+        self.emb = quantizer.modules[self.emb_node.target]
```

Comment:
these can happen in convert I think

Reply:
I see. Was following convention of other patterns in this file. Is there a benefit of moving this to convert?

Reply:
some of the other patterns need this because there are optional nodes in the pattern I think.
I think it's simpler to move this to convert, it's not necessary. to do this in __init__ in this case.